datafusion/physical-expr/src/aggregate/bit_and_or_xor.rs

```diff
@@ -751,6 +753,170 @@ impl RowAccumulator for BitXorRowAccumulator {
     }
 }
 
+/// Expression for a BIT_XOR(DISTINCT) aggregation.
+#[derive(Debug, Clone)]
+pub struct DistinctBitXor {
```

Comment:
I wonder if it is possible to avoid duplicated code for distinct aggregates -- they are all basically doing the same thing 🤔